src/Catty/PlayerEngine/AudioEngine/AudioPlayer.swift

```diff
@@ -98,7 +98,7 @@ class AudioPlayer: NSDiscardableContent {
     }
 
     func isPlaying() -> Bool {
```

Comment:
Could you please create computed properties for `isPlaying` and `isPaused` here that switch based on the player state?
```swift
var isPlaying: Bool {
    player.status == .playing
}

var isPaused: Bool {
    player.status == .paused
}
```
And then use them everywhere instead of other methods to get the state, this makes it easier to understand.
Also remove the current isPaused property and the isPlaying() function.